packages/flutter/lib/src/material/page.dart

```diff
@@ -267,12 +98,12 @@ class MaterialPageRoute<T> extends PageRoute<T> {
   Widget buildTransitions(BuildContext context, Animation<double> animation, Animation<double> forwardAnimation, Widget child) {
     if (Theme.of(context).platform == TargetPlatform.iOS &&
         Navigator.of(context).userGestureInProgress) {
-      return new _CupertinoPageTransition(
+      return new CupertinoPageTransition(
         animation: new AnimationMean(left: animation, right: forwardAnimation),
         child: child
```

Comment:
Oops, missed a comment. Wish github would just show review notes in chronological order. You're referring to this right? https://github.com/flutter/flutter/pull/9059#discussion_r108556485

I think I'm still favourable to just leaving holes in the route canvas and letting the user see into a shadow. But I think it won't 'flash' at the beginning of the transition since everything is offscreen for iOS and for Android opacity is 0. Was that what you meant?